### ISSUE TITLE
Turn Features Panel into a smaller column of icons

### ISSUE THREAD

**Issue by rodrigolzd** [conversejs/converse.js#1485]
I think the features panel is too large, perhaps the options could be to reduce it to a single column of icons on the far right. Take a look at the example link: https://imgur.com/mJC42Z1

That area on the bottom on the users would be more useful if i was seeing more users rather than showing me some list of features that i cant even click on. It's just informative text that takes up space. If there was something else, that we could interact with or some other information there i would be ok with the space being used, but not as it stands. 

If the column idea is not ideal, perhaps this icons could be put right under the room name on the top, without the text, but with the description showing up with a mouse hover.

I had posted this on Pade some time ago, and I advised to post it here instead
https://github.com/igniterealtime/Pade/issues/60

**Comment by jcbrand** [comment]
It's good to get UX feedback, but keep in mind that we don't have a descriptive icon for every feature and on mobile you can't hover your mouse to read about the feature.

For example the `moderated` and `non-moderated` features don't have descriptive icons.

So your suggestion will require more descriptive icons for certain features.

**Comment by jcbrand** [comment]
Closing as `wontfix` due to the reasons provided.

**Comment by ericschdt** [comment]
The "Features" panel takes a lot of space and at the same time only displays static content. 
There's already the **"i" icon** at the top panel, which shows all features if you press it. 

Hence, I'd really drop it for the following three reasons: 
1) You really look at it only once
2) It takes a lot of space
3) It makes the whole UI appear more cluttered and less clear
